dotnet/src/Connectors/VectorData.Abstractions/RecordDefinition/DistanceFunction.cs

```diff
@@ -25,7 +25,7 @@ public static class DistanceFunction
     public const string CosineSimilarity = nameof(CosineSimilarity);
 
     /// <summary>
-    /// The cosine (angular) similarity between two vectors.
+    /// Specifies the cosine (angular) distance between two vectors.
```

Comment:
Measures is probably more accurate here, since this is describing an algorithm for measuring similarity (or dissimilarity).```suggestion
    /// Measures the cosine (angular) distance between two vectors.
```